darwin-op/motion-manager.md

```diff
@@ -1,24 +1,24 @@
 ## Motion Manager
 
-This manager implement the `DARwInOPMotionManager` class and allows you to play
+The `DARwInOPMotionManager` class allows you to play
 a predefined motion stored in the "motion\_4096.bin" file. The main motions and
 their corresponding ids are listed in [appendix](motions-files.md).
 
 It is also possible to add custom motions to this file by using the `Action
 Editor` tool.
 
-> **note**:
-More informations about this tool provided by ROBOTIS is available on Robotis
+> **Note**:
```

Comment:
But in automobile, darwin-op, and reference books it seems that we always used the lower case.
I agree with David, it doesn't matter which case we use but it has to be consistent.
I will open a new issue about this.